nbs/docs/capabilities/08_cross_validation.ipynb

```diff
@@ -0,0 +1,813 @@
nftKyCWueQzlu2hTKLbKllr+b2f97P0oKENcdF8MLU77Ra1u3JEB17RTSUclHevRV48Eccv/EJblcOJhbaT8oldU0lFtKsMF8FeQVXV7FVxp5hFyX6w9fB9pVw6JtgyfG+ry+QyfsrYM5ps94JgwnJv/vJsOewqVXWnFr2D9PNayXcAinQCpbVqbQuKIH5RwKw5bmngTE64v6U6IHaz9jzN2l3sUnZ94atbLuoaXfzyVCiu0JxxfWkvde2PAS3X8aB+hzg9XLGC1DIUB7bRhspwnqrnGviEms1SfQZUfPq9trXvnbC/yuKsosPsqIogaq78/k8N9xwAx/72MdQFIV169bR39/Py1/+cudzwuEwJ554Ivfddx8XXXQRDz/8MIVCYcLnzJ8/n0MOOYT77ruvKomey+XI5VxtPuMBBC40URNKZdMhjbwopoWdS1OJviv8BIuCRaLf9dxgw8JFRzPWIaYW0j/WQFX03ggxNzrqmBuOD30D1P8JacGiQh1QolQ20SSqbgXke6IHS0qnnDZyfxveSrDoHmTnEmAYaq5YUaDIthwBuYegtGw/XhsVNY7kNeLJ3/PX5Pu4xziEiPF7aZeNGQHZucR72Eofefql27lEwyFAkdc2rOkkzDA5QrSF5AVYgShSWIRJOZ/G950oVuwwBJEj8/6aoTa+V3wVL1qxmMPMMrIaejP5EuvM+QBEOub7vl5Y11CNECVTQVPsUNyIPBJd2LlklTAhD+ruanjwgI/xpvVn8sauhZws44LlMkrZ2jdqwr+8TnROa+ci2RP9sDc5f03lLNtRv8V3NwQhL6XganeoZF1KdBnQVIV4SCMj1PIyPbABHv0FJ/U/wvXKJfK7lvKS3xVFER5oSHkXCyV6/1iWYqmMvuY2WPVnmHOQfxL98d9CdhT2f4XvIrxj55JwKdHHtsADP7AstU76lL+xCggSXa903IZ9kqeKTfJGzAy5Ylm6ollg1F6P/CrRe5QxDlA2oaQiwCESRubCZCW6jHthK9EjgkSX4THuCsXNCRJdVrdoPgkb7qG1c3/g9fL2Zevv4dg/vIkbQvvyeeM7cq7JFHZaTVRFzStFuVye8CcWi7FmzZoJHwvaHuWPf/wjo6OjvOMd7wCgv9/ym5ozZ86Ez5szZ47zb/39/YRCITo7O6t+zlS48soraW9vd/4sWhRAME8TNcH9IPtSojc90XeBIIrqDWwVWGaHizbKzmU0Xbty3lFnNu1cPMEh0evw2xNtYPlSmXxRso/lJFRa9f2qMCqb/KDCRWXZubQ2yM6lEmgmy86lGGj4cEqinUuQ7ws3cSzzYBWMJ7pox5arRI84HUKS1+V8khhZdEpSiZFoSONxcymrYsdAvHfmL6gD6UIw91j8/GmJhYq0D4/b6WB5zlsERiknz6fZrUSXSaKHo1G+WjyXB5ZcKMdH2EbWfl+GdFWafVJbJCTXb96FnWf+mLfnP8WANk9qUUV6Z1ipQsJpIa9KdOs9NjqlnYtkJboLaacIJG+edW+/i9ep9xDJD/m/2LKXwYGvJqH3AJK7qyJ6xRpFonUSUAnFJSyH1LMRC0nOdwCXJ7ohpaOmrzWMoSkUyyb941m5dkT/vgb+dgkMrfZdhO9z7Fxcz1V6yPoeD0vM37Pvr6lZ30+R4Duvhi1VcAvZYNTdj94AK39NJjkKQJdPEv2E9dfyj/CnOXzn3yQMbhLs+5soSlSih6wiRbgslOgSzpoTlOiSu0VD1nzQinbhWVawqKtLRWY3WODBznsRdntPdDd+/OMf88pXvpL58yeqNCZv4Gppn5npcz7zmc8wNjbm/Nm8ebP3gTfhC4LMiBgqIQ8vtyDb8/dkmKbpKETbPQaT7NtjvczW7UxKG1c1ZAslR8lZC7HrqDObdi6eIFQOHXVs0NyKqaAtXZI5OcGiYV1Ft0mLoMbsBItK8m8PnESXpJwXqm53N1EQEG2Hfkl/CDZYVLRxhjRVasdDEGtdWpIF0WTEPCrRU6kU8XiceDxOKjVF0dYmATKEpRYoYiGdbxbP5ptzvgrLT5Vz0WIe/noJb9j5PUIUpCvRl+VW8T3jKk5Yd42cC979Debc+XEOU9ZKJ9ENTSWnCBJdwj7CVm6ih5xCjdz5IL8IlEqlWD6/h63feh3nl2+G4XVSrtsWNSp+/jK7KIDxnhdwd/lwysb0tnr1Ii7bj7XoJtE9KtHtPdDwVHYuL7wILt0Gr5GgBCwV4Nm/w5p/Qrkk3xMd6L3vi1wV+i4Lipv8X+z0r8Cbb2AkvhSYulg147pdBS1hF4kuee4KgtsKdZaZ72Dbubi+h29M8ET3Pw9UVaHXzjnamcyDIbGTwkVEirnrlYic0zaFEl121wdYhOx+p5GdfxxgdVPUUhScbl4LJXpMyQbTlfuXj8Ef3kMhNQL4DxYVym61GIBP/sefgY+tYnvZErLK8USfSKLnpNi5VJToFZ9xSRSpPR/UgjVPpCnRRVaNacj1RG8q0WvGHkOib9y4kdtvv50LLrjA+djcuXMBdlGUDwwMOOr0uXPnks/nGRkZqfo5UyEcDtPW1jbhTxOzA8dyxKNaOsj2/D0ZuWKZfMkipb3Y5AAs6rI2NdtGJW9yp4CYB5qq1KTojctsW30eoh7Vv4CuqY6KI+gqth+LJzcURakEoga0aZCnRLd+F8lckXI5SGW3HNI/YmhO4TOooE5whcxKUKI7lkQBrBsZ2V6LNuIhyepNglOii0N11sP9TafTpNNTH/TKOfuAYoYCscrJFCTO33wSHvwhr0n/jjKKdN/5DsZ5hfYgixOPyLngmtuZs/Ym5itD0u19AG7RT+LF2WvY8dKv+r+YW4mel69Ej4V05jFEdPQ5qUGzmUyaYqHAxfwCBp+Vcs22iO6o/GUTkUKtJ9uqYNHYg/zU+BqvG/qRnAva5EjJVAiHvCk2HU/0qexcjIhFQqkS7kN2HH75Frjh9WCaTiFT5vogfOH1cp5CSU63YOW9NvU9mG7droaWiBGMjRY49j
```

Comment:
Throughout the tutorial there's use of 'time step' and 'timestep', maybe make that consistent